cloud-common/src/main/resources/application.yml

```diff
@@ -147,27 +153,39 @@ cb:
             repoid: HDP-2.5
             redhat6: http://public-repo-1.hortonworks.com/HDP/centos6/2.x/updates/2.5.5.0
             redhat7: http://public-repo-1.hortonworks.com/HDP/centos7/2.x/updates/2.5.5.0
-            repository-version: 2.5.3.0-37
-            vdf-redhat6: http://public-repo-1.hortonworks.com/HDP/centos6/2.x/updates/2.5.3.0/HDP-2.5.3.0-37.xml
-            vdf-redhat7: http://public-repo-1.hortonworks.com/HDP/centos7/2.x/updates/2.5.3.0/HDP-2.5.3.0-37.xml
+            debian9: http://public-repo-1.hortonworks.com/HDP/debian7/2.x/updates/2.5.5.0
+            ubuntu16: http://public-repo-1.hortonworks.com/HDP/ubuntu16/2.x/updates/2.5.5.0
+            repository-version: 2.5.5.0-157
+            vdf-redhat6: http://public-repo-1.hortonworks.com/HDP/centos6/2.x/updates/2.5.5.0/HDP-2.5.5.0-157.xml
+            vdf-redhat7: http://public-repo-1.hortonworks.com/HDP/centos7/2.x/updates/2.5.5.0/HDP-2.5.5.0-157.xml
+            vdf-debian9: http://public-repo-1.hortonworks.com/HDP/debian7/2.x/updates/2.5.5.0/HDP-2.5.5.0-157.xml
+            vdf-ubuntu16: http://public-repo-1.hortonworks.com/HDP/ubuntu16/2.x/updates/2.5.5.0/HDP-2.5.5.0-157.xml
           util:
             repoid: HDP-UTILS-1.1.0.21
             redhat6: http://public-repo-1.hortonworks.com/HDP-UTILS-1.1.0.21/repos/centos6
             redhat7: http://public-repo-1.hortonworks.com/HDP-UTILS-1.1.0.21/repos/centos7
+            debian9: http://public-repo-1.hortonworks.com/HDP-UTILS-1.1.0.21/repos/debian7
+            ubuntu16: http://public-repo-1.hortonworks.com/HDP-UTILS-1.1.0.21/repos/ubuntu16
       2.6:
-        version: 2.6.4.5-2
+        version: 2.6.4.1-30
```

Comment:
do we need to change the default hdp version?

Reply:
For Ubuntu16 and Debian9 that version is not available:
```
$ aws s3 ls s3://private-repo-1.hortonworks.com/HDP/debian7/2.x/updates/2.6.4
                           PRE 2.6.4.1-30/
$ aws s3 ls s3://private-repo-1.hortonworks.com/HDP/ubuntu16/2.x/updates/2.6.4
                           PRE 2.6.4.1-30/
```

Reply:
In that case we need to involve @akanto, because I'm not sure about the desired version in 2.4

Reply:
We've talked about changing the HDP version and unfortunately we cannot do it because of the long process of the certification. Could you please send this PR to branch rc-2.5 without the hdp version changes. I know it will be invalid for Ubuntu and Debian, but for now you can overcome this by providing the version on the UI or CLI to the correct one. Once we merge back the rc-2.5 to master we will include newer versions of Ambari and HDP and get this aligned.

Reply:
Done